tagstudio/src/qt/widgets/item_thumb.py

```diff
@@ -194,7 +195,15 @@ def __init__(
         self.opener = FileOpenerHelper("")
         open_file_action = QAction("Open file", self)
         open_file_action.triggered.connect(self.opener.open_file)
-        open_explorer_action = QAction("Open file in explorer", self)
+        
+        system = platform.system()
+        if system == "Darwin":
+            open_explorer_action = QAction("Reveal file in Finder", self)
+        elif system == "Linux":
+            open_explorer_action = QAction("Open file in explorer", self) # TODO: Rename to whatever the Linux explorer is
```

Comment:
you could make this say "Reveal in filesystem"

Reply:
For Linux or for all of them? Because I think it's nice to have it say Finder on Mac (plus most apps do that I think)

Reply:
Just Linux

Reply:
k should be fixed now